plugins/wpe-headless/phpunit.xml.dist

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<phpunit
+	bootstrap="tests/bootstrap.php"
+	backupGlobals="false"
+	colors="true"
+	convertErrorsToExceptions="true"
+	convertNoticesToExceptions="true"
+	convertWarningsToExceptions="true"
+>
+	<testsuites>
+		<testsuite>
```

Comment:
```suggestion
		<testsuite name="integration">
```

phpunit 7.x complains if the name attribute isn't set here.

```
PHPUnit 7.5.20 by Sebastian Bergmann and contributors.

  Warning - The configuration file did not pass validation!
  The following problems have been detected:

  Line 11:
  - Element 'testsuite': The attribute 'name' is required but missing.

  Test results may not be as expected.
```

Reply:
Odd. I am not seeing that. Maybe I need a rebuild.